setup.py

```diff
@@ -9,7 +9,7 @@
 requires = [
     'pyramid',
     'crabpy>=0.15.0',
-    'pycountry>=18.12.8'
+    'pycountry>=23.12.11'
```

Comment:
kweeni of we dit niet beter gewoon vastpinnen als ze nog is van iso veranderen gaat niet ineens alles kapot gaan. Anderzijds worden we dan wel verplicht mee up te daten. Kies maar :) voor de rest ok